### ISSUE TITLE
Add a third "Tab Indicator" differentiating between "Current-Focused Tab" and "Current-Unfocused Tab"  

### ISSUE THREAD

**Issue by Yaron10** [notepad-plus-plus/notepad-plus-plus#8834]
**STR:**
Enable `Draw a colored bar on active tab`.
Give keyboard focus to a "not-editor" element (e.g. the Incremental Search Bar).

**Result:**
The `Active tab focused indicator` is used even when the tab *does not have keyboard focus*.

**Suggested:**
Three tab indicators.
**1.** "Current-Focused Tab".
**2.** "Current-Unfocused Tab".
**3.** "Current Tab in Other View".

-- The user would be able to instantly know if the editing window actually has keyboard focus.
-- The system color for "Active Window Title" could be set to "Current-Focused Tab".

Related: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8446.

***

Notepad++ v7.8.9   (32-bit)
Build time : Sep  5 2020 - 02:32:39
Path : C:\---\notepad++.exe
Admin mode : ON
Local Conf mode : ON
OS Name : Windows 7 Enterprise (32-bit) 
OS Build : 7601.23964
Current ANSI codepage : 1255
Plugins : ComparePlus.dll HTMLTag_unicode.dll MenuIcons.dll PythonScript.dll _CustomizeToolbar.dll

**Comment by KnIfER** [comment]
I prefer a "Marked Tab Indicator" that indicates the file is prominent.`
